content/ko/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -93,7 +93,7 @@ metadata:
 spec:
   containers:
   - name: agnhost
-    image: k8s.gcr.io/e2e-test-images/agnhost:2.35
+    image: registry.k8s.io/e2e-test-images/agnhost:2.35
```

Comment:
```suggestion
    # 이미지가 변경됨 (기존에는 "k8s.gcr.io" 레지스트리를 사용)
    image: registry.k8s.io/e2e-test-images/agnhost:2.35
```

Reply:
Thank you for the suggestion! I applied it and squashed commits, please take a look.